community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: tidb-operator.1.0.0-beta1
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+        [  
+          {
+            "apiVersion": "pingcap.com/v1alpha1",
+            "kind": "TidbCluster",
+            "metadata": {
+              "generation": "1",
+              "labels": {
```

Comment:
I don't think these labels are necessary, and may be added by the cluster on creation.